README.rst

```diff
@@ -62,38 +62,47 @@ them, please visit the following URLs:
 
 -  Announcements: http://lists.idyll.org/listinfo/khmer-announce
 
-We chat at https://gitter.im/dib-lab/khmer and the maintainers can be
-contacted at khmer-project@idyll.org.
+We have a slack chat that [you can join](LINK/EMAIL/JOIN) and the maintainers
```

Comment:
Yeah, see Titus' comment. Issue tracker for support, with a brief mention of the mailing list.